### DISCUSSION TITLE
Support skip locked in postgresql.

### DISCUSSION THREAD

**Discussion by meotimdihia** [prisma/prisma#4786] [Q&A] [answered]
This is an important feature for queue jobs. 

https://www.2ndquadrant.com/en/blog/what-is-select-skip-locked-for-in-postgresql-9-5/

It's hard to develop without this feature in medium or large projects.

**Comment by nikolasburk** [comment, marked as answer]
Hey @meotimdihia, if you'd like to see native support for skip locked in Prisma it would be best if you could [open a new feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) that describes your use case for it in more detail. That way, our product and engineering teams can look into it and incorporate it in our [roadmap](https://www.notion.so/prismaio/Prisma-Roadmap-50766227b779464ab98899accb98295f).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
